### PR TITLE
Update README, daily build docs, and contributing guide

### PR DESCRIPTION
**Pull request by natemcmaster**
Preparation for merging this with aspnet/AspNetCore. I took the content of that repo and merged it with the README's and docs of this repo.

For easier review, see https://github.com/aspnet/Universe/tree/namc/readme

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+4. See error
+
+### Expected behavior
+A clear and concise description of what you expected to happen.
+
+### Screenshots
+If applicable, add screenshots to help explain your problem.
+
+### Additional context
+Add any other context about the problem here.
```
Comment (Eilon) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Maybe add the suggestion

> Show the output of `dotnet --info`

Or whatever?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,4 +1,73 @@
...
+
+## Other discussions
+Our team members also monitor several other discussion forums:
+
+* [ASP.NET Core forum](https://forums.asp.net/1255.aspx/1?ASP+NET+5)
+* [Stack Overflow](https://stackoverflow.com/) with the [`asp.net-core`](https://stackoverflow.com/questions/tagged/asp.net-core), [`asp.net-core-mvc`](https://stackoverflow.com/questions/tagged/asp.net-core-mvc), or [`entity-framework-core`](https://stackoverflow.com/questions/tagged/entity-framework-core) tags.
+
+
+## Filing issues
+When filing issues, please use our [bug filing templates](https://github.com/aspnet/AspNetCore/wiki/Functional-bug-template).
```
Comment (Eilon) [inline CONTRIBUTING.md]: Do you want to remove (or heavily edit) this section, since you're adding proper issue templates?

File: README.md [hunk 3]
```diff
@@ -1,122 +1,45 @@
...
-`TestOnly`                      | Don't package or verify things.                                                | `/p:TestOnly=true`
-`KOREBUILD_REPOSITORY_INCLUDE` | A list of the repositories to include in build (instead of all of them).       | `$env:KOREBUILD_REPOSITORY_INCLUDE='Antiforgery;CORS'`
-`KOREBUILD_REPOSITORY_EXCLUDE` | A list of the repositories to exclude from build (all the rest will be built). | `$env:KOREBUILD_REPOSITORY_EXCLUDE='EntityFramework'`
-
-## More info
-
-This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+Follow the [Getting Started](https://docs.microsoft.com/aspnet/core/getting-started) instructions in the [ASP.NET Core docs](https://docs.microsoft.com/aspnet/index).
+
+Also checkout the [.NET Homepage](https://www.microsoft.com/net) for released versions of .NET, getting started guides, and learning resources.
```
Comment (Eilon) [inline README.md]: Since you're here: `checkout` --> `check out`. Nouns vs. verbs. When will people learn 😄

File: README.md [hunk 4]
```diff
@@ -1,122 +1,45 @@
...
+
+* [Documentation](https://github.com/aspnet/Docs) - documentation sources for https://docs.microsoft.com/aspnet/core/
+* [EntityFrameworkCore](https://github.com/aspnet/EntityFrameworkCore) - data access technology
+* [Tooling](https://github.com/aspnet/Tooling) - Visual Studio tooling, editors, and dialogs
+
+## Code of conduct
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).  For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## License
```
Comment (Eilon) [inline README.md]: This section isn't needed.

**Comment by natemcmaster** [comment]
:up: :date:

File: CONTRIBUTING.md (hunk 2 above)
Comment (natemcmaster) [reply CONTRIBUTING.md]: Sounds good to me. Removed this section in favor of issue templates.

**Comment by Eilon** [comment]
Looks great!

**Pull request merged**